### PR TITLE
skeleton for tracking popup

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers

### CODE REVIEW

File: react-native/react/tracker/action-render.desktop.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+'use strict'
+/* @flow */
+
+import React from '../base-react'
+import BaseComponent from '../base-component'
+
+export default class ActionRender extends BaseComponent {
+  constructor (props) {
```
Comment (MarcoPolo) [inline react-native/react/tracker/action-render.desktop.js]: :scissors:

Comment (MarcoPolo) [inline react-native/react/tracker/action-render.desktop.js]: and same for all the other empty constructors

**Comment by MarcoPolo** [comment]
:+1: % C

File: react-native/react/tracker/action-render.desktop.js (hunk 1 above)
Comment (cjb) [inline react-native/react/tracker/action-render.desktop.js]: I'm happy to do a project wide removal of these, feel free to merge in and leave it to me.

Comment (chrisnojima) [inline react-native/react/tracker/action-render.desktop.js]: I'll strip them

**Pull request merged**